### ISSUE TITLE
Improve the visibility of the educational resources page

### ISSUE THREAD

**Issue by dmint789** [thewca/worldcubeassociation.org#8902]
**Is your feature request related to a problem? Please describe.**
The [educational resources](https://www.worldcubeassociation.org/education) section of the website is useful for competitors and organizers alike, and will become even more useful as the WQAC continues to add to it, but it is very much unknown to most users of the website, because it's not found in the most obvious place. Currently, you have to go to the Information dropdown, and the vast majority of users don't know the link to the educational resources is there, or that that page even exists.

**Describe the solution you'd like**
I would like to see the following additons:

1. A new section on the home page in addition to the current 5 boxes there called "Educational Resources" with a "Learn more" button and maybe the following description (may edit this later):

New to competing in the WCA? Check out our resources for new competitors that explain the whole process. You can also learn about organizing your own WCA competition in great detail!

2. A blue/light blue information box that is shown on the registration page when the competitor successfully registers, with the following text:

Check out the [Educational Resources](https://www.worldcubeassociation.org/education) page that explains the process of competing in a WCA competiton!

3. A blue/light blue information box that is shown at the top of the competition creation form with the following text:

Have a look at the [Educational Resources](https://www.worldcubeassociation.org/education) page that has many resources for organizing WCA competitions.

**Comment by dunkOnIT** [comment]
I agree with the core problem - that we don't surface information well, especially for newcomers. 

Another thought this spawns is to split "Educational Resources" into "Newcomer/Competitor Resources" and "Organizer Resources"

Definitely happy to accept PR's against (2) and (3), for (1) I feel like it may call for some more design/UX thinking around how to present info to newcomers in a concise, compelling way - some kind of "I want to compete" or "How to compete" button that takes people to a more concise page with an overview of the process. 

It may be worth getting WMT involved on that as well perhaps.

**Comment by dmint789** [comment]
I think it's okay to leave it without splitting it, as there is already a separate organizer guidelines page, and some of the resources on the main educational resources page could be used by competitors too (e.g. a competitor may want to print out covers to use at home).

Regarding (1), I'm happy to get in touch with the WMT.

**Comment by dmint789** [comment]
@dunkOnIT we have discussed it, and the WMT will consider if they want to work on an actual redesign of the WCA home page (relevant to 1.). Could the WST implement 2. and 3. for now?

**Comment by dmint789** [comment]
@dunkOnIT kindly pinging :)

**Comment by dunkOnIT** [comment]
Hi Deni! Apologies again for the delay on this. Discussed the changes with Gregor and we decided the following. Basically we want to balance making information available to those who need it without adding too much clutter to what are already-busy pages.

* Regarding (2), we have added the message as requested **for all newcomers** - see screenshot below
![image](https://github.com/thewca/worldcubeassociation.org/assets/52967253/8ed4b3f4-044c-4e89-b767-1f2d7fdb94e3)

* Regarding (3), we've opted not to add a message to the competition form for two reasons:
  * The Competition Form is already cluttered/features a lot of message boxes, and we don't want to add to the noise
  * Only a very small set of people using the competition form will actually need the educational resources - the vast majority are either experienced organizers/delegates, or are being mentored by experienced organizers into region-specific practices. If you have alternative suggestions for where we could place this link that would target new organizers specifically, feel free to suggest it.

**Comment by dmint789** [comment]
@dunkOnIT Thanks a lot for making that change!

Regarding the other change, I don't think that only a small percentage of users of the competition form will find it useful, as most documents on the educational resources page are aimed at organizers. There are 14 organizer guidelines documents, the competition templates with a lot of useful stuff, and the certificates made by the WCT. Also, the judge tutorial and the competitor tutorial are useful for organizers, who often share these with competitors. We've also seen many delegates/organizers recreate some of these tutorials, not knowing about the educational resources. We believe this leads to a lot of wasted effort, and potentially also inconsistencies between regions. I understand there is already a lot on that page, but we believe it would be a great benefit if organizers and delegates were made aware of the existence of these resources somewhere in the competition creation process. I would like to strongly ask the WST to reconsider. I've seen too many delegates pleasantly surprised by the fact this page even exists, and it feels like the work the WQAC does could do more good if we just made people aware of the resources we've created.

**Comment by dmint789** [comment]
@dunkOnIT any thoughts on this?

**Comment by dmint789** [comment]
@thewca/software-team can someone please have a look at this? I'd really like to see (3) added too.

**Comment by gregorbg** [comment]
Re (3), we usually use blue/light blue boxes to communicate technical information like "This page has unsaved changes".

So we definitely need to use a different color for the box. Why does it have to be a colored box anyways? The only people who are able to reach the Create Competition page at any time under any circumstances are Delegates. They already know a lot about organizing. Then, when they deliberately add some user account as organizer, these select organizers gain access as well. But the Delegate has to make the conscious choice of adding them, and at least as far as I am concerned, they will only do so when they are confident that the person they are adding has sufficient knowledge about organizing.

What problem are you trying to solve by adding an "advertisement" to your resources to an already crammed and complicated form?

**Comment by gregorbg** [comment]
As a compromise, what about adding a section about educational resources to the "You have been added to [XYZ Open 2024] as an organizer" notification email?

**Comment by dmint789** [comment]
> So we definitely need to use a different color for the box. Why does it have to be a colored box anyways? The only people who are able to reach the Create Competition page at any time under any circumstances are Delegates. They already know a lot about organizing. Then, when they deliberately add some user account as organizer, these select organizers gain access as well. But the Delegate has to make the conscious choice of adding them, and at least as far as I am concerned, they will only do so when they are confident that the person they are adding has sufficient knowledge about organizing.

I understand, but I will refer to my previous point in reply to Duncan above:

> We've also seen many delegates/organizers recreate some of these tutorials, not knowing about the educational resources. We believe this leads to a lot of wasted effort, and potentially also inconsistencies between regions.

This is exactly the problem we are trying to solve with this "advertisement". If you can think of a better place on the website for this, I'm open to suggestions, although I'd much rather it were somewhere on the website and not in an email 99% of people don't read.

**Comment by dmint789** [comment]
@gregorbg @dunkOnIT any further thoughts?
